### PR TITLE
FileViewer: ShowSyntaxHighlighting hotkey and context menu

### PR DESCRIPTION
**Pull request by gerhardol**
## Proposed changes

Hotkey for ShowSyntaxHighlighting  to quickly deactivate highlighting in reviews, conflicting with other colors. 
Especially useful with #11464 that adds more colors.

Menu item is for completeness and to display current hotkey.

Sorting context menu as toolbar buttons

Mnemonics  in context menu

Default hotkey for ShowSyntaxHighlighting   and ShowEntireFile

### Before

image

### After

image

## Test methodology 

Manual

## Merge strategy

I agree that the maintainer squash merge this PR (if the commit message is clear).

----

:black_nib: I contribute this code under [The Developer Certificate of Origin](../blob/master/contributors.txt).

### CODE REVIEW

**Comment by mstv** [comment]
Should we sort the context menu items in order to match the sequence of the toolbar?

image

**Comment by gerhardol** [comment]
> Should we sort the context menu items in order to match the sequence of the toolbar?

I considered that too. Can do it this PR if agreed.

File: GitUI/Translation/English.xlf [hunk 1]
```diff
@@ -1610,6 +1610,10 @@ The primary difftool can still be selected by clicking the main menu entry.</sou
         <source>Show syntax highlighting</source>
         <target />
       </trans-unit>
+      <trans-unit id="showSyntaxHighlightingToolStripMenuItem.Text">
+        <source>Show syntax highlighting</source>
```
Comment (mstv) [inline GitUI/Translation/English.xlf:1614]: Mnemonics would be good, too. :)

Comment (gerhardol) [reply GitUI/Translation/English.xlf]: Updated, see note at updated screenshot

File: GitUI/Hotkey/HotkeySettingsManager.cs [hunk 2]
```diff
@@ -331,6 +331,7 @@ public IReadOnlyList<HotkeySettings> CreateDefaultSettings()
                     Hk(FileViewer.Command.NextChange, Keys.Alt | Keys.Down),
                     Hk(FileViewer.Command.PreviousChange, Keys.Alt | Keys.Up),
                     Hk(FileViewer.Command.ShowEntireFile, Keys.None),
+                    Hk(FileViewer.Command.ShowSyntaxHighlighting, Keys.None),
```
Comment (mstv) [inline GitUI/Hotkey/HotkeySettingsManager.cs:334]: ❔
```suggestion
                    Hk(FileViewer.Command.ShowEntireFile, Keys.Ctrl + Keys.E),
                    Hk(FileViewer.Command.ShowSyntaxHighlighting, Keys.X),
```

**Review by mstv — COMMENTED** [review]
```
a WS amount
b
c Copy
d Decrease
e Show entire file
f Find
g Goto
h Non-pr chars
i Increase
j
k
l WS EOL
m
n Copy new
o Copy old
p Copy patch
q
r Reset
s Stage
t Text
u Unstage
v
w WS
x Syntax highlighting
y
z
```

**Comment by RussKie** [comment]
:shipit:

**Review by mstv — APPROVED** [review]
Thank you

**Pull request merged**